src/chrome/chromeDebugAdapter.ts

```diff
@@ -567,7 +600,7 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
 
         this._expectingStopReason = undefined;
 
-        smartStepP.then(should => {
+        await smartStepP.then(should => {
```

Comment:
I add it to include the time spent in this operation in the overall measurement of onPaused event. Apart from the measurement function, no one seems to actively wait on the promise returned from this method.